mm2src/mm2_test_helpers/dummy_files/iris_test_history.json

```diff
@@ -0,0 +1,170 @@
+        "received_by_me": "0",
+        "my_balance_change": "-0.028456",
+        "block_height": 6148318,
+        "timestamp": 0,
+        "fee_details": {
+            "type": "Utxo",
+            "coin": "IRIS-TEST",
+            "amount": "0.028455"
+        },
+        "coin": "IRIS-TEST",
+        "internal_id": "4135353530353436314138373531433631443333364331460000000000000000",
+        "transaction_type": "StandardTransfer"
+    },
+    {
+        "tx_hex": "0a2a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a7261301240304544424136304632423642383739324133333739423232453445323144373243343145413530464436353331433243383445413135353937303834373432391a4034353138383334373164316536383461323334663362323063626638646137393338636330643038383562626237326266303837363465386431356536353138",
+        "tx_hash": "10D92256317083E49F06A9DD2184714DEEE9C55A505CFEF037BE8E87E9FB82ED",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "total_amount": "0.01",
+        "spent_by_me": "0",
+        "received_by_me": "0.01",
+        "my_balance_change": "0.01",
+        "block_height": 6148311,
+        "timestamp": 0,
+        "fee_details": {
+            "type": "Utxo",
+            "coin": "IRIS-TEST",
+            "amount": "0.02648"
+        },
+        "coin": "IRIS-TEST",
+        "internal_id": "4444394136304639344533383037313336353232394430310000000000000000",
+        "transaction_type": "StandardTransfer"
+    },
+    {
+        "tx_hex": "0a2a696161316530727838376d646a37397a656a65777563346a6737716c3975643232383667327573386632122a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a7261301a0e0a036e696d1207313030303030301a100a05756e79616e120731303030303030",
+        "tx_hash": "3057F075161ECD149F4F5A629F852D5E7E57411DECE7AD35CC7D11E7EB06E0BF",
+        "from": [
+            "iaa1e0rx87mdj79zejewuc4jg7ql9ud2286g2us8f2"
+        ],
+        "to": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "total_amount": "1",
+        "spent_by_me": "0",
+        "received_by_me": "1",
+        "my_balance_change": "1",
+        "block_height": 6148039,
+        "timestamp": 0,
+        "fee_details": {
+            "type": "Utxo",
+            "coin": "IRIS-TEST",
+            "amount": "0.026861"
+        },
+        "coin": "IRIS-TEST",
+        "internal_id": "3236413546344639343144434531363135373046373530330000000000000001",
+        "transaction_type": "StandardTransfer"
+    },
+    {
+        "tx_hex": "0a2a696161316530727838376d646a37397a656a65777563346a6737716c3975643232383667327573386632122a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a7261301a0e0a036e696d1207313030303030301a100a05756e79616e120731303030303030",
+        "tx_hash": "3057F075161ECD149F4F5A629F852D5E7E57411DECE7AD35CC7D11E7EB06E0BF",
```

Comment:
As two transfers of the same asset were made during a single tx, it might be worth to sum them up with `"my_balance_change":"2"`.